com.valvesoftware.Steam.Utility.steamtinkerlaunch.yml

```diff
@@ -0,0 +1,63 @@
+id: com.valvesoftware.Steam.Utility.steamtinkerlaunch
+default-branch: stable
+sdk: org.freedesktop.Sdk//21.08
+runtime: com.valvesoftware.Steam
+runtime-version: stable
+build-extension: true
+appstream-compose: false
+separate-locales: false
+rename-icon: steamtinkerlaunch
+rename-desktop-file: steamtinkerlaunch.desktop
+
+build-options:
+  prefix: /app/utils/steamtinkerlaunch
+  prepend-path: /app/utils/steamtinkerlaunch/bin
+  append-pkg-config-path: /app/utils/steamtinkerlaunch/lib/pkgconfig
+  strip: true
+
+cleanup:
+  - /include
+  - /lib/*.a
+  - /lib/*.la
+  - /share/doc
+  - /share/man
+
+modules:
+  - modules/git.yml
+  - modules/innoextract.yml
+  - modules/jq.yml
+  - modules/xdotool.yml
+  - modules/xprop.yml
+  - modules/xxd.yml
+  - modules/xwininfo.yml
+  - modules/yad.yml
+
+  - name: steamtinkerlaunch
+    build-options:
+      env:
+        PREFIX: /app/utils/steamtinkerlaunch
+    buildsystem: simple
+    build-commands:
+      - make install
+      - install -Dm644 -t ${FLATPAK_DEST}/share/steam/compatibilitytools.d SteamTinkerLaunch.vdf
+      - install -Dm644 -t ${FLATPAK_DEST}/bin toolmanifest.vdf
```

Comment:
Nitpick: installing non-executable files to `$bindir` isn't very nice. Maybe put `toolmanifest.vdf` to extension mount point root (then I guess we'll need to remove `bin/` prefix from `SteamTinkerLaunch.vdf` and add it to `toolmanifest.vdf` instead)?

Reply:
The Valve compat tools documentation state that the `toolmanfiest.vdf` should be next to the tool being lanched. Unless I understood them wrong?

Reply:
If "next to" means "in the same directory", then I wonder if compat tools are supposed to be installed to system `PATH` at all.